openmdao/docs/advanced_guide/implicit_comps/defining_icomps.rst

```diff
@@ -107,7 +108,7 @@ The Newton solver's use of that Jacobian information is why we need to declare a
 
 .. embed-code::
     openmdao.test_suite.test_examples.test_circuit_analysis.TestCircuit.test_circuit_plain_newton
-    :layout: code, output
+    :layout: interleave
```

Comment:
why did you switch these to interleave?

Reply:
to make it easier for the reader to distinguish the solver output and identify which values were which (it was a large block of numbers), and it didn't effect the contiguousness of the code itself, just the print statements.